Cargo.toml

```diff
@@ -33,6 +33,7 @@ tokio-util = "0.7.10"
 toml = "0.8"
 tracing = "0.1.40"
 tracing-subscriber = { version = "0.3.18", default-features = false, features = [
+    "ansi",
```

Comment:
Just my $0.02, it feels like this should be set in [`wkg` Cargo.toml](https://github.com/bytecodealliance/wasm-pkg-tools/blob/main/crates/wkg/Cargo.toml#L26) and not at the workspace level.